### ISSUE TITLE
RADIUS filters : merge_request is ignored

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#1012]
Using the following configuration : 

```
[dindemaster]
filter = ssid
operator = is
value = Dindemaster

[vlan2:dindemaster]
scope = returnRadiusAccessAccept
merge_request = no
answer1 = Tunnel-Medium-Type => 6
answer2 = Tunnel-Type => 13
answer3 = Tunnel-Private-Group-ID => 2
```

Gives the following response : 

```
Wed Nov 11 08:33:33 2015 : Debug: Sending Access-Accept packet to host 172.20.110.250 port 32769, id=44, length=0
Wed Nov 11 08:33:33 2015 : Debug:       Reply-Message = "PacketFence has proceed the request"
Wed Nov 11 08:33:33 2015 : Debug:       Tunnel-Private-Group-Id:0 = " 2"
Wed Nov 11 08:33:33 2015 : Debug:       User-Name = "94:db:c9:38:85:5b"
Wed Nov 11 08:33:33 2015 : Debug:       Tunnel-Medium-Type:0 = IEEE-802
Wed Nov 11 08:33:33 2015 : Debug:       Tunnel-Type:0 = VLAN
Wed Nov 11 08:33:33 2015 : Debug:       Airespace-ACL-Name = "ok"
```

Note that Airespace-ACL-Name shouldn't be there

**Comment by julsemaan** [comment]
Switch conf : 

```
[a.b.c.d]
mode=production
defaultRole=ok
AccessListMap=N
description=sd
registrationRole=portal
gamingRole=ok
type=Cisco::WLC_http
portalURL=http://a.b.c.d
VoIPEnabled=N
guestRole=ok
isolationRole=portal
radiusSecret=radius
goldenRole=ok
guestVlan=20
defaultVlan=20
goldenVlan=20
gamingVlan=20
isolationVlan=20
registrationVlan=20
VlanMap=N
```

**Comment by julsemaan** [comment]
fixed by af0152e8d7b8e3aa24aa6b1561624ba43aed8fdb
